test/unit/org/apache/cassandra/tools/StandaloneVerifierTest.java

```diff
@@ -43,12 +54,143 @@ public void testStandaloneVerifier_NoArgs()
     }
 
     @Test
-    public void testStandaloneVerifier_WithArgs()
+    public void testMaybeChangeDocs()
     {
-        runner.invokeClassAsTool("org.apache.cassandra.tools.StandaloneVerifier", "--debug", "system_schema", "tables")
-              .waitAndAssertOnCleanExit();
-        assertNoUnexpectedThreadsStarted(EXPECTED_THREADS_WITH_SCHEMA, OPTIONAL_THREADS_WITH_SCHEMA);
-        assertSchemaLoaded();
-        assertServerNotLoaded();
+        // If you added, modified options or help, please update docs if necessary
+        try (ToolRunner tool = runner.invokeClassAsTool(StandaloneVerifier.class.getName(), "-h"))
+        {
+            String help = "usage: sstableverify [options] <keyspace> <column_family>\n" + 
+                           "--\n" + 
+                           "Verify the sstable for the provided table.\n" + 
+                           "--\n" + 
+                           "Options are:\n" + 
+                           " -c,--check_version          make sure sstables are the latest version\n" + 
+                           "    --debug                  display stack traces\n" + 
+                           " -e,--extended               extended verification\n" + 
+                           " -h,--help                   display this help message\n" + 
+                           " -q,--quick                  do a quick check, don't read all data\n" + 
+                           " -r,--mutate_repair_status   don't mutate repair status\n" + 
+                           " -t,--token_range <range>    long token range of the format left,right.\n" + 
+                           "                             This may be provided multiple times to define multiple different ranges\n" + 
+                           " -v,--verbose                verbose output\n";
+            Assertions.assertThat(tool.getStdout()).isEqualTo(help);
+        }
+    }
+
+    @Test
+    public void testWrongArgFailsAndPrintsHelp()
+    {
+        try (ToolRunner tool = runner.invokeClassAsTool(StandaloneVerifier.class.getName(), "--debugwrong", "system_schema", "tables"))
+        {
+            assertThat(tool.getStdout(), CoreMatchers.containsStringIgnoringCase("usage:"));
+            assertThat(tool.getCleanedStderr(), CoreMatchers.containsStringIgnoringCase("Unrecognized option"));
+            assertEquals(1,tool.getExitCode());
+        }
+    }
+
+    @Test
+    public void testDefaultCall()
+    {
+        try (ToolRunner tool = runner.invokeClassAsTool(StandaloneVerifier.class.getName(), "system_schema", "tables"))
+        {
+            assertThat(tool.getStdout(), CoreMatchers.containsStringIgnoringCase("using the following options"));
+            assertTrue(tool.getCleanedStderr(), tool.getCleanedStderr().isEmpty());
```

Comment:
I changed it but what garbage are you referring to? Like too much output?

Reply:
we capture the bytes, then toString it, then call `java.lang.String#replaceAll` for each pre-defined regex (which itself recompiles the Pattern).

Might not be a lot of cost, but its a simple pattern to avoid so was mostly calling it out so we can avoid moving forward.

Reply:
Ok icwym now. Thx.